experimental/dds/tree2/src/test/feature-libraries/defaultChangeFamily.spec.ts

```diff
@@ -83,15 +84,15 @@ function initializeEditableForest(data?: JsonableTree): {
 		initializeForest(forest, [singleTextCursor(data)]);
 	}
 	let currentRevision = mintRevisionTag();
-	const repairStore = new ForestRepairDataStore(forest);
+	const repairStore = new ForestRepairDataStore(forest, defaultIntoDelta);
 	const changes: TaggedChange<DefaultChangeset>[] = [];
 	const deltas: Delta.Root[] = [];
 	const builder = new DefaultEditBuilder(
 		family,
 		(change) => {
 			changes.push({ revision: currentRevision, change });
+			repairStore.capture(change, currentRevision);
```

Comment:
This is new - was it a bug to not do this before?

Reply:
This was just moved up a couple lines. I probably did it during the delta -> change switch.